modules/src/applications/transfer/relay/on_ack_packet.rs

```diff
@@ -0,0 +1,19 @@
+use crate::applications::transfer::acknowledgement::Acknowledgement;
+use crate::applications::transfer::context::Ics20Context;
+use crate::applications::transfer::error::Error as Ics20Error;
+use crate::applications::transfer::packet::PacketData;
+use crate::applications::transfer::relay::refund_packet_token;
+use crate::core::ics04_channel::packet::Packet;
+
+pub fn process_ack_packet(
```

Comment:
This is a fairly short function called only once, we could just inline the implementation in `on_acknowledgement_packet()`?